common/xash3d_types.h

```diff
@@ -120,7 +120,7 @@ typedef uint64_t longtime_t;
 #define LittleLongSW(x) (x = LittleLong(x) )
 #define LittleShort(x) ((short)( (((short)(x) >> 8) & 255) + (((short)(x) & 255) << 8)))
 #define LittleShortSW(x) (x = LittleShort(x) )
-_inline float LittleFloat( float f )
+static inline float LittleFloat( float f )
```

Comment:
You can include `port.h` where needed:
https://github.com/FWGS/xash3d-fwgs/blob/master/common/port.h#L37=